CONFIGURE.md

```diff
@@ -214,4 +215,11 @@ At the moment, libjade only supports Linux and Darwin based operating systems on
 
 Note: `ALG` in `OQS_ENABLE_LIBJADE_KEM_ALG/OQS_ENABLE_LIBJADE_SIG_ALG` should be replaced with the specific algorithm name as demonstrated in OQS_ENABLE_KEM_ALG/OQS_ENABLE_SIG_ALG.
 
-**Default**: `OFF` if OQS_LIBJADE_BUILD is `OFF` else unset.
\ No newline at end of file
+**Default**: `OFF` if OQS_LIBJADE_BUILD is `OFF` else unset.
+
+## OQS_BUILD_FUZZ_TESTS
+Can be `ON` or `OFF`. When `ON` liboqs the fuzz test-suite will be enabled. This option is only available if `$CC` is set to clang.
```

Comment:
Nit: I would rather direct users to the `CMAKE_C_COMPILER` option than the `CC` environment variable.

Reply:
I think I've got this right now, but please confirm this is what you are after as I'm not a cmake wizard.